src/NHibernate.Test/NHSpecificTest/GH1879/ExpansionRegressionTests.cs

```diff
@@ -30,35 +21,6 @@ protected override void OnSetUp()
 			}
 		}
 
-		protected override void Configure(Configuration configuration)
```

Comment:
This, unfortunately, makes the test useless.

Reply:
It actually looks wrong that user can't use own generator for already registered method.
I think we should simply change code to allow user overrides (basically change `dictionary.Add(method, handler)` with `dictionary[method] = handler` in affected places)

Reply:
i agree, some method handlers could be already defined by some developers (like in our case :-)) what could leads to exceptions, should i rewrite the code and revert test

Reply:
> should i rewrite the code and revert test

I would say yes. Please do.